docs/how-tos/scale-applications-using-appgw-metrics.md

```diff
@@ -0,0 +1,107 @@
+# Scale your Applications using Application Gateway Metrics (Beta)
+
+As incoming traffic increases, it becomes crucial to scale up your applications based on the demand.
+
+In the following tutorial, we explain how you can use Application Gateway's `AvgRequestCountPerHealthyHost` metric to scale up your application. `AvgRequestCountPerHealthyHost` is measure of average request that are sent to a specific backend pool and backend http setting combination.
+
+We are going to use following two components:
+
+1. [`Azure K8S Metric Adapter`](https://github.com/Azure/azure-k8s-metrics-adapter) - We will using the metric adapter to expose Application Gateway metrics through the metric server.
+1. [`Horizontal Pod Autoscaler`](https://docs.microsoft.com/en-us/azure/aks/concepts-scale#horizontal-pod-autoscaler) - We will use HPA to use Application Gateway metrics and target a deployment for scaling.
+
+## Setting up Azure K8S Metric Adapter
+
+1. We will first create an Azure AAD service principal and assign it `Monitoring Reader` access over Application Gateway's resource group.
```

Comment:
We need to explain what one should do with the lines of code here. It is not clear that it is `bash`.
I'd append something like - ...paste this in a bash cloud shell...